tests/settings_tests/tests.py

```diff
@@ -286,6 +287,41 @@ def test_override_settings_nested(self):
         with self.assertRaises(AttributeError):
             getattr(settings, 'TEST2')
 
+    def test_no_secret_key(self):
+        with self.assertRaisesMessage(ImproperlyConfigured, 'The SECRET_KEY setting must not be empty.'):
+            os.environ[ENVIRONMENT_VARIABLE] = 'test'
+            settings._setup()
+
+    def test_no_setings_module(self):
```

Comment:
*settings

also since this is making two assertions, i would make it two separate test methods